extras/lbf-nix/lbf-rust.nix

```diff
@@ -78,13 +78,12 @@ let
       '';
     };
 
-  # 
   crateVersions = pkgs.writeTextFile {
     name = "lambda-buffers-crate-versions";
     text = ''
       num-bigint = "0.4.4"
       serde_json = { version = "1.0.107", features = ["arbitrary_precision"] }
-      plutus-ledger-api = { git = "https://github.com/mlabs-haskell/plutus-ledger-api-rust", features = [ "lbf", ], rev = "fb93fa590908580eb40368369bf6614d42ce9a95" }
+      plutus-ledger-api = { git = "https://github.com/mlabs-haskell/plutus-ledger-api-rust", features = [ "lbf", ], ref = "szg251/fix-lbf", rev = "4460f2ff8af7993a24209ebe28df5ef9b3b3b397" }
```

Comment:
It's going to be tricky managing updates across the board. Perhaps we should document and try to use tags for these foundational deps?

Reply:
Yes, I want to improve this situation...